index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="rwapper">
+      <header>
+        <nav>
+          <p class="mylogo">My Logo</p>
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </header>
+      <section class="main">
+        <main>
+          <h1>
+            I'm Emmanuel Glad to see <br />
+            you!
+          </h1>
+          <p class="txt">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <p class="connect">LET'S CONNECT</p>
+        </main>
+        <footer>
+          <ul class="social">
+            <li>
+              <a href="#"><img src="img/Twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="img/Linkedin icon.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="img/MVector.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="img/Github.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="img/PeaceVector.png" alt="" /></a>
+            </li>
+          </ul>
+        </footer>
+      </section>
```

Comment:
Please consider making your code semantically correct by using these semantic tags when necessary and at the right place.

### Further clarification:-

Your code needs to look like that. REMEMBER I just replaced and made it semantically correct by using the tags correctly else the markup is the same. If you want to learn more about this topic consider reviewing them. [Article 1](https://web.dev/learn/html/semantic-html/)
[Article 2](https://www.w3schools.com/html/html5_semantic_elements.asp)

```
 <main class="main">
      
      <section>
        <h1>
          I'm Emmanuel Glad to see <br />
          you!
        </h1>
        <p class="txt">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
        <a href="/" class="connect">LET'S CONNECT</a>

        <ul class="social">
          <li>
            <a href="#"><img src="img/Twitter.png" alt="" /></a>
          </li>
          <li>
            <a href="#"><img src="img/Linkedin icon.png" alt="" /></a>
          </li>
          <li>
            <a href="#"><img src="img/MVector.png" alt="" /></a>
          </li>
          <li>
            <a href="#"><img src="img/Github.png" alt="" /></a>
          </li>
          <li>
            <a href=""><img src="img/PeaceVector.png" alt="" /></a>
          </li>
        </ul>
      </section>
    </main>

```

Reply:
Done